.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: python
 python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
 
 env:
-  - PLONE_VERSION=4.3 QA=true
+- PLONE_VERSION=4.3
+- PLONE_VERSION=5.0
```

Comment:
@hvelarde Wouldn't it make more sense to not test Plone 5 at all? It will fail and just take up time.
I'd move Plone 5 testing into a separate branch, make the tests work in Plone 5 and merge that into master.